app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+from typing import Any, Optional
+
+from app.errors import (
+NotVaccinatedError,
+OutdatedVaccineError,
+NotWearingMaskError
+)
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict[str, Any]) -> Optional[str]:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("The visitor is not vaccinated.")
```

Comment:
Code Style Issue: You are raising a `NotVaccinatedError` without a descriptive error message. It would be better to include a descriptive error message when raising an exception. For instance, you can change it to `raise NotVaccinatedError("The visitor is not vaccinated.")`.